scripts/beachball/index.ts

```diff
@@ -9,6 +9,11 @@ export const config: BeachballConfig = {
       include: ['packages/office-ui-fabric-react', 'packages/react'],
       disallowedChangeTypes: ['major'],
     },
+    {
+      name: 'Charting',
+      include: ['packages/charting', 'stub-packages/react-charting'],
```

Comment:
Was this intentionally meant to be `react-charting` or should this be `style-utilities`?